app/src/main/java/com/example/playlistmaker/SearchActivity.kt

```diff
@@ -121,6 +148,58 @@ class SearchActivity : AppCompatActivity() {
         }
     }
 
+    private fun updateVisibiltyViews(
+        noConnection: Boolean = false, empty: Boolean = false, hideList: Boolean = false
```

Comment:
Если в функции больше 1 параметра - принято делать каждый с новой строки
```suggestion
        noConnection: Boolean = false, 
        empty: Boolean = false, 
        hideList: Boolean = false
```